cli/src/commands/operation/undo.rs

```diff
@@ -75,6 +75,20 @@ pub fn cmd_op_undo(
         let template = tx.base_workspace_helper().operation_summary_template();
         template.format(&bad_op, formatter.as_mut())?;
         writeln!(formatter)?;
+
+        let was_undo_op = *tx.repo().view() == parent_op.view()?;
+        if args.operation == "@" && was_undo_op {
```

Comment:
This is a review to discuss the `op undo` vs `undo` question, but I think at this point we're fairly decided: `op undo` and `undo` now behave identically, and both will display the hint.

The hint will still not be displayed if the user attempts to undo a specific operation (e.g. `jj [op] undo 1234abc`), with the assumption that they know what they're doing in that case.

Reply:
I don't think there's much left to discuss, before either embarking on a new `jj undo` command (#3700), as both Martin and Yuya are fine with the current solutio.